### DISCUSSION TITLE
pointer-events: none not getting removed from body after dialog close.

### DISCUSSION THREAD

**Discussion by raviteja83** [radix-ui/primitives#1748] [Help] [answered]
Dialog is opened on clicking an item from dropdown menu. Dialog is conditionally rendered with defaultOpen prop and close state is updated with `onOpenChange`

`@radix-ui/react-dialog@1.0.2`,
`@radix-ui/react-dropdown-menu@1.0.0`

Do let me know if you need any other versions or anything else. Seems to have started happening recently.

**Comment by benoitgrelard** [comment, marked as answer]
It's probably due to versions. Make sure you update to the latest version of both packages as they need to be aligned.
If not, try to reproduce the issue in a sandbox and we can take a look.

**Reply by raviteja83** [reply]
Sure, let me try with the updated versions.
